test/test.sh

```diff
@@ -100,7 +100,8 @@ for arch in x86 x64 arm arm64; do
     EXEC "" BIN=$BIN ./test-meson.sh
 
     # MSBuild requires .NET framework v4.x or Mono to run.
-    if [[   -d /usr/share/wine/mono \
+    if [[ $OSTYPE == darwin* \
```

Comment:
Could we, instead of this, look for the mono installation in whatever location it is on macOS?

Reply:
Searching the .Net Framework assembly `System.dll` in the whole system:
```shell
$ find / -name "System.dll" 2>/dev/null`
/Applications/Wine Staging.app/Contents/Resources/wine/share/wine/mono/wine-mono-8.1.0/lib/mono/4.8-api/System.dll
...
```
Mono only exists under `/Applications/Wine Staging.app`. So I think the condition `$OSTYPE == darwin*` is good enough.

Reply:
That’s kinda surprising. So the macOS GitHub runners do have wine staging installed there, in /Applications - while we use the main wine that we install ourselves with homebrew - and that finds the mono parts from within another random app bundle? 

In the earlier attempts by @ravenexp , he tried installing wine-mono with homebrew, but didn’t get that setup working.

Reply:
> So the macOS GitHub runners do have wine staging installed there
No, the test was done on my local VM. Installing `wine-stable` via brew will be at `/Applications/Wine Stable.app`.

Reply:
In that case, can you add a comment explaining that on macOS, wine (and wine-mono) can be installed in a variety of locations, e.g. under /Applications in an application bundle, so we don't quite know where to look for it - so we just assume that it is available and run this test?

Reply:
done.